code/game/machinery/autolathe.dm

```diff
@@ -201,14 +201,19 @@
 		// open the new ui window
 		ui.open()
 
-
 /obj/machinery/autolathe/attackby(var/obj/item/I, var/mob/user)
 	if(default_deconstruction(I, user))
 		return
 
 	if(default_part_replacement(I, user))
 		return
 
+	if(istype(I, /obj/item/weapon/disk/autolathe_disk))
```

Comment:
Recycling disks should be allowed in the case that
1. The disk has/had limited uses
and
2. Its uses have been depleted to zero

Reply:
those cases are taken into account